File: CS/ExtendedDataNavigator/Navigator/ExtendedDataNavigatorButtons.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;

namespace VerticalDataNavigator.Navigator
{
    public class ExtendedDataNavigatorButtons : ControlNavigatorButtons
    {
        public ExtendedDataNavigatorButtons(INavigatorOwner owner)
            : base(owner) {}

        protected override NavigatorButtonsViewInfo CreateViewInfo()
        { return new ExtendedDataNavigatorButtonsViewInfo(this); }

    }
}

File: VB/ExtendedDataNavigator/Navigator/ExtendedDataNavigatorButtons.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo

Namespace VerticalDataNavigator.Navigator
	Public Class ExtendedDataNavigatorButtons
		Inherits ControlNavigatorButtons

		Public Sub New(ByVal owner As INavigatorOwner)
			MyBase.New(owner)
		End Sub

		Protected Overrides Function CreateViewInfo() As NavigatorButtonsViewInfo
			Return New ExtendedDataNavigatorButtonsViewInfo(Me)
		End Function

	End Class
End Namespace
